apis/placement/v1beta1/clusterresourceplacement_types.go

```diff
@@ -488,6 +488,10 @@ const (
 	// RollingUpdateRolloutStrategyType replaces the old placed resource using rolling update
 	// i.e. gradually create the new one while replace the old ones.
 	RollingUpdateRolloutStrategyType RolloutStrategyType = "RollingUpdate"
+
+	// StageRollingUpdateRolloutStrategyType replaces the old placed resource stage by stage
```

Comment:
Hi Ryan! Infosys mentioned that this would apply to first-time placement as well.